docs/GettingStartedDocs/Contributors/AdvancedBuildInfo.md

```diff
@@ -12,6 +12,8 @@ cd openenclave/scripts/ansible
 ansible localhost -m import_role -a "name=linux/openenclave tasks_from=environment-setup.yml" --become --ask-become-pass
 ```
 
+Note: The Ansible scripts only support configuring clean SGX-enabled machines, or any machines already configured with these Ansible scripts.
```

Comment:
Consider also adding a link to a doc for non-clean machines.

Reply:
@jiria I have included a link for non-clean machines as well. Please see the updated pull request diff.